ChangeLog

```diff
@@ -13,6 +13,16 @@
 
 2015-06-14  Titus Brown  <titus@idyll.org>
 
+   * scripts/extract-paired-reads.py: added --output_dir, --paired-output,
```

Comment:
ok shall I make this mergable/fix ChangeLog, or are there other things that
need to be done?